src/main/java/net/snowflake/client/loader/ProcessQueue.java

```diff
@@ -89,8 +91,8 @@ public void run() {
           }
           // Create a temporary table to hold all uploaded data
 
-          int loaded = 0;
-          int parsed = 0;
+          long loaded = 0;
+          long parsed = 0;
           int errorCount = 0;
```

Comment:
What about making `errorCount` to `long`, does it also have the same issue with changing errorRecordCount from int to long?

Reply:
errorCount should be okay! It doesn't get its value from any getInt or getLong functions; rather, it starts at 0 and increments by 1 when there's an error. So I think it can stay as an int.